.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           name: Check code
           command: |-
             curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | TMPDIR=$(mktemp -d) sh -s -- -b $(go env GOPATH)/bin v1.15.0
-            make -j$(sysctl -n hw.physicalcpu) -C ./builddir check
+            make -j$(sysctl -n hw.physicalcpu) -C ./builddir vendor-check check
```

Comment:
Why is this `make` call using `hw.physicalcpu` instead of `nproc` ? Is there a reason for it?

Reply:
@jmstover `nproc` does not exist on the mac platform

Reply:
@jmstover @mem I see that `sysctl` has a `hw.logicalcpu` name. That may be more reasonable to use since it includes logical core count from hyperthreading. What do you think?

Reply:
that would match nproc, which gives you the CPU count from the point of view of the OS.

`getconf _NPROCESSORS_ONLN` gives you the number of online processors, which should be equal or larger than the number reported by nproc. That should work on OSX, too.

- hw.physicalcpu :-1:
- hw.logicalcpu :+1: 
- getconf _NPROCESSORS_ONLN :+1: